wordpress-mobile/WordPress-iOS
# Fix Noto font family not loaded correctly

Fixes #19877
Refs p5T066-3ND-p2#comment-14170, p5T066-3ND-p2#comment-14172

> **Warning**: This depends on https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/331. Once that PR is merged, the pod version of `WordPressShared` should be updated before merging this PR.

This fixes the issue of the Noto font family not loading correctly due to incorrect access to the resource bundle from the WordPressShared library.

## To test:

- Run the app.
- Open any comment threads.
- 🔍 Verify that the comment bodies are displayed with the Noto font family.
- Open the editor.
- 🔍 Verify that the title and body are displayed with the Noto font family.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the changes.

3. What automated tests I added (or what prevented me from doing so)
N/A.

PR submission checklist:
